native/cocos/bindings/event/EventDispatcher.cpp

```diff
@@ -383,17 +407,37 @@ void EventDispatcher::dispatchCloseEvent() {
 void EventDispatcher::dispatchDestroyWindowEvent() {
 #if CC_PLATFORM == CC_PLATFORM_WINDOWS
     EventDispatcher::dispatchCustomEvent(EVENT_DESTROY_WINDOW, 1,
-                                         reinterpret_cast<void *>(CC_GET_PLATFORM_INTERFACE(ISystemWindow)->getWindowHandle()));
+                                         reinterpret_cast<void *>(CC_GET_MAIN_SYSTEM_WINDOW()->getWindowHandle()));
+#else
+    EventDispatcher::dispatchCustomEvent(EVENT_DESTROY_WINDOW, 0);
```

Comment:
window id is always 0 on platforms except windows?

Reply:
I did not modify it, I am not sure too.

Reply:
the 2nd parameter of `dispatchCustomEvent` is the number of args, 0 means no parameter.

Reply:
no argument? Where to check the value of the parameter?